src/App.js

```diff
@@ -1,5 +1,34 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import styled from "styled-components";
+
+import MainHeader from "./components/MainHeader";
+
 function App() {
```

Comment:
code style 변경해주시면 좋을 것 같습니다!
```suggestion
export default function App() {
```